src/Compilers/Test/Core/Assert/ConditionalFactAttribute.cs

```diff
@@ -222,9 +222,21 @@ public override bool ShouldSkip
 
     public class IsEnglishLocal : ExecutionCondition
     {
-        public override bool ShouldSkip =>
-            !CultureInfo.CurrentUICulture.Name.StartsWith("en", StringComparison.OrdinalIgnoreCase) ||
-            !CultureInfo.CurrentCulture.Name.StartsWith("en", StringComparison.OrdinalIgnoreCase);
+        public override bool ShouldSkip
+        {
+            get
+            {
+                // WSL environments can have this value as empty string
```

Comment:
were you missing some english-only tests on linux locally?

Reply:
Yes. Several of the VB rebuild tests were english only and I couldn't run them on my WSL environment but they were failing in CI. This unblocked me.